microsoft/PowerToys
# [settings-ui] Settings WinUI3

## Summary of the Pull Request

**What is this about:**
 - Move Settings to WinUI3 (WinAppSdk 1.0.1)
 - Always run settings non-elevated (WinAppSdk 1.0.1 does not support WinUI3 unpackaged up elevation)
 - Move everything to .NET6
 - Force update all files on reinstall/update (REINSTALLMODE) - should fix .png files (etc...) missing issues on update


NOTES:
 - PowerLauncherPage.xaml and ShortcutDialogContentControl.xaml - InfoBadge not available - marked TODO: - not blocking
 - @crutkas publish.cmd file, Settings part, is it needed. Removed here.

**What is included in the PR:** 

**How does someone test / validate:** 
- Build installer and install PT
- Open Settings and OOBE
- CLICK EVERY BUTTON IN SETTINGS AND OOBE! and confirm it's working properly
- Test show oobe and show scoobe scenarios

## Quality Checklist


## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
